Exploration Queries/SecurityEvent/ParentProcessesOnHost.txt

```diff
@@ -1,20 +1,18 @@
-// Name: Parent Processes running on Host
+// Id: 992b9912-a11e-487d-83b3-1d1ba6081746
+// DisplayName: Parent Processes running on Host
 // Description: Any 4688 event that contains Parent Process running on a given host during the time of or recently after compromise and in the range of +-10m
```

Comment:
you have removed the timeframe variables so should adjust the comment/description accordingly

Reply:
the timeframe was removed from the query but would still be applied by us when running the query.
but I think you meant that when copying the query and executing it externally it would not longer make sense
will change it accordingly